fastlane_app.rb

```diff
@@ -48,6 +49,7 @@ class FastlaneApp < Sinatra::Base
     # display of real-time output
     set(:server, "thin")
     if ENV["WEB_APP"]
+      use(FastlaneCI::JwtAuth)
```

Comment:
does this run on every route? I think we just want this on our API calls right?

Reply:
As it is inside the `ENV["WEB_APP"]`, it will only be loaded for the Angular application that middleware (to be tested, tho)

Reply:
You're right, this should be included in every endpoint controller that uses `JwtAuth` instead of there, my bad.

Reply:
Maybe `AuthenticatedControllerBase` is not useful for our API endpoints, and we should inherit from a custom `JWTAuthenticatedControllerBase` that uses the middleware.

Reply:
I think ideally that would be the case, but switching to that enforcement right now would break our UI, since we do not have the token being sent yet. We could move the Authentication controller base work into another CL, with the Token verification and keep that open until the frontend is complete.

Reply:
Yep, I just want to make sure that the `/login` endpoint generates the required token, even if we don't we use it for now. Later on, we can apply the middleware into the controllers when the frontend is finished. Is that ok?

Reply:
Fine with me. I'll let one of the ruby experts give you the approval 😀